### PR TITLE
Cleanup spec file

### PR DESCRIPTION
**Pull request by smarlowucf**
Change to multispec with python 2 disabled.

### CODE REVIEW

File: csp-billing-adapter.spec [hunk 1]
```diff
@@ -15,28 +15,35 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
-
+%{?!python_module:%define python_module() python-%{**} python3-%{**}}
```
Comment (rtamalin) [inline csp-billing-adapter.spec:18]: Shouldn't this be provided by the standard rpm macros used by the build?

File: csp-billing-adapter.spec [hunk 2]
```diff
@@ -15,28 +15,35 @@
...
-BuildRequires:  python3-pytest-cov
+BuildRequires:  %{python_module pytest}
+BuildRequires:  %{python_module coverage}
+BuildRequires:  %{python_module pytest-cov}
 %endif
-Requires:       python3-pluggy
-Requires:       python3-python-dateutil
+Requires:       %{python_module setuptools}
+Requires:       %{python_module pluggy}
+Requires:       %{python_module python-dateutil}
```
Comment (rtamalin) [inline csp-billing-adapter.spec:40]: Don't see a requires for PyYAML

File: csp-billing-adapter.spec (hunk 1 above)
Comment (smarlowucf) [reply csp-billing-adapter.spec:18]: I don't really know the state of python multispec. This is the line I see in the packages I've looked at for reference. And it seems to work as expected :shrug: .

File: csp-billing-adapter.spec (hunk 2 above)
Comment (smarlowucf) [reply csp-billing-adapter.spec:41]: Thanks, added

**Review by brett060102 — APPROVED** [review]
LGMT

File: csp-billing-adapter.spec (hunk 1 above)
Comment (smarlowucf) [reply csp-billing-adapter.spec:18]: Leaving as is for now.

Comment (rtamalin) [reply csp-billing-adapter.spec:18]: Yeah that is fine... And the line has no effect if the macro is already defined...

**Pull request merged**